### ISSUE TITLE
Microphone Gain Moves to 100% in Android App

### ISSUE THREAD

**Issue by austinseraphin** [BearWare/TeamTalk5#748]
Sometimes, without warning, the microphone gain will suddenly move to 100%. Trying to adjust it with the buttons does not work. I use TalkBack. I have to close the app and open it again, then bring the gain back down to the default of 50%.

**Comment by bear101** [comment]
Please provide a way to reproduce the issue and then reopen.
